### ISSUE TITLE
[ROS2 Gem Robot Importer] Failure during conversion and exporting a ```.dae``` file.

### ISSUE THREAD

**Issue by Mechazo11** [o3de/o3de-extras#752]
**Describe the bug**
Hello. I am trying to import a robot whose mesh and urdf files are available in a ros2 package. This ros2 package has been built successfully in a workspace and I can use Ignition to open the robot model correctly with all of its mesh loaded correctly.

However, when I use the ```Robot Importer``` tool, all urdf and mesh files are loaded correctly except one ```.dae``` file.  For ease of explanation, lets call this part ```partA.dae```. It uses COLLADA XML schema.

```O3DE Asset Processor``` wizard shows the following error 
![image](https://github.com/user-attachments/assets/5b0e14ad-470b-42fa-a09d-51640dee060f)

Is it required to rebuild the ```partA.dae``` from its 3D model or does the Robot Importer tool have a feature to correct this issue?

**Expected behavior**
All urdf and mesh files are loaded correctly by the ```Robot Importer``` tool and a ```robot_name.prefab``` is created. This ```prefab``` then loads the robot model correctly in the level and later can be made into its own stand-alone GEM.

**Actual behavior**
Asset Processor throws error. The ```robot_name.prefab``` generates a robot entity with the mesh for ```partA.dae``` missing. The robot's model is erratic.

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS & Version: Ubuntu 22.04 Jammy Jellyfish
 - CPU Ryzen 5600X
 - GPU RTX 3080
 - Memory 16GB + 16 GB

**Additional context**
Unable to share actual .dae file at this time due to a timed-embargo.

With best,
@Mechazo11

**Comment by Mechazo11** [comment]
I solved this problem using ```Blender 4.2```.

Apparently the problem was with the ```cover_03``` component in the ```partA.dae``` file which may have had an ill-formed mesh. I am not sure, but at the time of writing this comment, ```Blender 4.2``` does not have the ```Mesh Analyzer``` (probably I was not able to activate it following the tutorials) activated by default. Hence I was not able to correct the mesh at this time.

I ended up delete the problematic component from the ```.dae``` file and re-exporting ```partA.dae```. After that I updated the ```full_robot.xacro``` (the master XACRO file for the whole robot) to accept the new ```.dae``` file to solve the above mentioned problem.

Hope this helps.
With best,
@Mechazo11
